stdlib/public/Windows/CMakeLists.txt

```diff
@@ -4,5 +4,8 @@ add_swift_target_library(swiftWinSDK ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_SDK_
     SWIFT_COMPILE_FLAGS ${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS} ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
     LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
     TARGET_SDKS WINDOWS
-    INSTALL_IN_COMPONENT sdk-overlay)
-
+    INSTALL_IN_COMPONENT sdk-overlay
+    # Workaround for the issue swiftWinSDK.dll in the built directory might be being used
```

Comment:
This sounds like a build-system architectural issue. Nothing that we just-built should be running. Will keep my eyes open for this as I continue pushing through the build. 🤔

Reply:
Yes. This is a horrible workaround. But I haven't been able to find a way to avoid it.